lib/elixir/unicode/tokenizer.ex

```diff
@@ -4,40 +4,45 @@ defmodule String.Tokenizer do
   data_path = Path.join(__DIR__, "UnicodeData.txt")
 
   {letter_uptitlecase, start, continue, _} =
-    Enum.reduce File.stream!(data_path), {[], [], [], nil}, fn
-      line, {letter_uptitlecase, start, continue, first} ->
-        [codepoint, line] = :binary.split(line, ";")
-        [name, line] = :binary.split(line, ";")
-        [category, _] = :binary.split(line, ";")
-
-        {codepoints, first} =
-          case name do
-            "<" <> _ when is_integer(first) ->
-              last = String.to_integer(codepoint, 16)
-              {Enum.to_list(last..first), nil}
-            "<" <> _ ->
-              first = String.to_integer(codepoint, 16)
-              {[first], first + 1}
-            _ ->
-              {[String.to_integer(codepoint, 16)], nil}
-          end
-
-        cond do
-          category in ~w(Lu Lt) ->
-            {codepoints ++ letter_uptitlecase, start, continue, first}
-          category in ~w(Ll Lm Lo Nl) ->
-            {letter_uptitlecase, codepoints ++ start, continue, first}
-          category in ~w(Mn Mc Nd Pc) ->
-            {letter_uptitlecase, start, codepoints ++ continue, first}
-          true ->
-            {letter_uptitlecase, start, continue, first}
+    Enum.reduce(File.stream!(data_path), {[], [], [], nil}, fn line,
+       {letter_uptitlecase, start, continue, first} ->
```

Comment:
Can you please rewrite this as:

```elixir
Enum.reduce(File.stream!(data_path), {[], [], [], nil}, fn line, acc ->
  {letter_uptitlecase, start, continue, first} = acc
```

This will remove the weird line break (which is also fixed in the formatter in master).

Reply:
yes. I should have caught that.